.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["@babel/preset-env"],
+  "plugins": ["@babel/plugin-transform-modules-commonjs"]
```

Comment:
@THEBOSS0369 Don't transform modules to CJS format, that will lead to loads of issues.